### ISSUE TITLE
Duplicate tag

### ISSUE THREAD

**Issue by Liaune** [kingdido999/atogatari#7]
https://atogatari.com/screenshot/58fea902b2d83a26e170d1d0
General tag & Anime tag are both eva，can't convert the general eva to anime

**Comment by kingdido999** [comment]
Yes, I'm aware of this issue, which is caused by extra space not being trimmed (in this case, "eva "). I have already made the fix to both client side and server side.

I will come up a strategy to merge existing tags...
